bib/internal/setup/setup.go

```diff
@@ -53,6 +53,14 @@ func EnsureEnvironment() error {
 	if err := util.RunCmdSync("mount", "--bind", destPath, osbuildPath); err != nil {
 		return err
 	}
+
+	// Ensure we have devfs inside the container to get dynamic loop
+	// loop devices inside the container.
+	devMnt := "/dev/"
+	if err := util.RunCmdSync("mount", "-t", "devtmpfs", "devtmpfs", devMnt); err != nil {
```

Comment:
In bootc's similar logic I structured things to be idempotent, because we need to do this for the case of running with `--entrypoint bash` and invoking the CLI multiple times.

Of course, this *will* work today, we'll just end up with a new mount for each invocation.

We'll mostly get that by moving this invocation above the bind mount, which the function uses as an "I already succeeded" marker.

Reply:
Eh, I was wrong; in the success case it will work fine.  But it still seems slightly better to leave the bind mount as an "I succeeded" marker and not do anything after; perhaps add a comment like:

```
// NOTE: Don't add new code here, do it before the bind mount which acts as the final success indicator
```

Reply:
Thank you, that is an excellent suggestion. I pushed to https://github.com/osbuild/bootc-image-builder/pull/238 as it seems to be fine to land without this PR. I hope I understood you suggestion correctly.